From 7d460400702a323c3dc796f95ce5e155767c0aca Mon Sep 17 00:00:00 2001
From: Torsten Beyer <tb@pobox.com>
Date: Sun, 28 Aug 2022 20:04:45 +0200
Subject: [PATCH] Set minimum CPU voltage to 1.2V

The cubieboard2 dev spec says minimum voltage for lower frequencies should be 1.0iV and 1.1V respectively. Usage of cubieboards in OpenVarion
show instable behaviour with voltages of 1.0V and 1.1V. Hence this change increases CPU voltage to 1.2V for the lowest CPU speeds.
---
 arch/arm/boot/dts/sun7i-a20.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/sun7i-a20.dtsi b/arch/arm/boot/dts/sun7i-a20.dtsi
index 5574299685ab..6902621d806f 100644
--- a/arch/arm/boot/dts/sun7i-a20.dtsi
+++ b/arch/arm/boot/dts/sun7i-a20.dtsi
@@ -112,9 +112,9 @@
 				<912000	1400000>,
 				<864000	1300000>,
 				<720000	1200000>,
-				<528000	1100000>,
-				<312000	1000000>,
-				<144000	1000000>;
+				<528000	1200000>,
+				<312000	1200000>,
+				<144000	1200000>;
 			#cooling-cells = <2>;
 		};
 
@@ -130,9 +130,9 @@
 				<912000	1400000>,
 				<864000	1300000>,
 				<720000	1200000>,
-				<528000	1100000>,
-				<312000	1000000>,
-				<144000	1000000>;
+				<528000	1200000>,
+				<312000	1200000>,
+				<144000	1200000>;
 			#cooling-cells = <2>;
 		};
 	};
-- 
2.15.0

